samcli/hook_packages/terraform/Config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "hook_package_id": "terraform",
+    "description": "Terraform IaC hooks package",
+    "hook_use_case": "IaC",
+    "iac_framework": "Terraform",
+    "version": "1.0",
+    "hook_specification": "1.0.0",
+    "functionalities": {
+         "prepare": {
+              "entry_script": "./prepare/main"
```

Comment:
probably question to @moelasmar, do we need `parameters` here?

Reply:
I believe we're not supporting parameters for now